configs/webpack-config-compass/src/index.ts

```diff
@@ -102,6 +102,17 @@ const sharedResolveOptions = (
         'index.js'
       ),
 
+      // TODO: document what's going on here
+      '@leafygreen-ui/button/constants': path.resolve(
```

Comment:
This is a dependency of lg-chat for a single (!) re-exported value that is used in one (!) place that doesn't exist in our version of `@leafygreen-ui/button`. While we can override it for webpack, this still causes issues for unit tests because those are not bundled (and we wouldn't want to start bundling them). This is probably something we should ask leafygreen team to adjust for us and hardcode it in place where it's used because there is no real way for us to deal with this otherwise (like maybe we can also start using patch-package, but there should be some limit to the amount of hacks we're introducing)

Reply:
After playing around with it, patching is the only way forward, but hopefully this is short lived